packages/localize-tools/testdata/build-runtime-xliff/input/locale-codes.ts

```diff
@@ -0,0 +1,25 @@
+// Do not modify this file by hand!
```

Comment:
This looks like the output is somehow getting checked out into the input folder? Same for a bunch of other files.